### PR TITLE
#6 adding monitoring and health check endpoint

### PR DESCRIPTION
**Pull request by maxim-tschumak**

### CODE REVIEW

File: crawler/build.gradle [hunk 1]
```diff
@@ -36,6 +36,7 @@ repositories {
 
 dependencies {
     compile('org.springframework.boot:spring-boot-starter')
+    compile("org.springframework.boot:spring-boot-starter-web")
```
Comment (fmueller) [inline crawler/build.gradle]: Minor: I think with starter-web in place you can remove starter.

**Comment by fmueller** [comment]
👍

File: crawler/build.gradle (hunk 1 above)
Comment (maxim-tschumak) [inline crawler/build.gradle]: Thanks, I've removed the unused dependency.

**Comment by fmueller** [comment]
👍

**Pull request merged**